galaxy/api/internal/views/combined.py

```diff
@@ -115,3 +116,58 @@ def _validate_order(self, order):
                     fields=str(allowed_orders)
                 )
             )
+
+
+class CombinedDetail(base.APIView):
+    '''
+    This is intendended to provide all of the information for the content
+    detail pages. For repos, it returns the repository, namespace and list of
+    content items.
+
+    For collections it returns a collection object
+    '''
+    def get(self, request):
+        namespace = request.GET.get('namespace', None)
```

Comment:
Question. How are we handling case sensitivity. Specifically, thinking about #1810.

Reply:
Just reviewed #1824. Do we potentially have the same issue here, where we need to convert `-` to `_` before performing the lookup?

Reply:
Case is insensitive, I think it could make sense to incorporate the `-` 👉`_` bit, if you want to basically let github URL paths work here for consistency, but does that make sense for collections where they don't have to match repository names at all?

Reply:
I think making the query case insensitive is fine.

I don't think we should perform a `-` -> `_` conversion. We still have a lot of namespaces with `-` in the name and a change like this would make all of those pages impossible to find.